pkgs/tools/misc/farge/default.nix

```diff
@@ -24,14 +25,24 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-vCMuFMGcI4D4EzbSsXeNGKNS6nBFkfTcAmSzb9UMArc=";
   };
 
-  buildInputs = [ bash bc feh imagemagick ]
-    ++ lib.optionals waylandSupport [ grim slurp wl-clipboard ]
-    ++ lib.optionals x11Support [ xcolor ];
+  nativeBuildInputs = [ makeBinaryWrapper ];
+
+  # Ensure the following programs are found within $PATH
+  wrapperPath = with lib; makeBinPath ([
```

Comment:
Remove the `with lib;`, it's not really needed.

Reply:
Done